contrib/helm/event-gateway/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: serverless/event-gateway
-  tag: 0.9.0
+  tag: latest
```

Comment:
I think we should revert this change

Reply:
Why? latest is set to v0.9.1...you don't want people deploying latest EG?

Reply:
Nevermind. It's fine :D